### PR TITLE
update to UK Driver's License pattern

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/548

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/what-the-sensitive-information-types-look-for.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/what-the-sensitive-information-types-look-for.md]:
```suggestion
- Five digits in the date format MMDDY for date of birth (7th character is incremented by 50 if driver is female, i.e. 51 to 62 instead of 01 to 12)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@VLG17 - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. This is based on issue #548

Thank you.

cc: @kenwith

**Comment by chrfox** [comment]
contacting PG to confirm

**Comment by chrfox** [comment]
second request sent to PG (vaibhav/mas) to confirm, waiting on response

**Pull request merged**
